File: Logitude/AngularModules/AngularModules/Common/EntityPMs/AgentSharedLogisticsKey.ts




import {UIProperties, UIProperty} from '../../Infrastructure/Components/LogitudeComponents/UIProperties';
import {ServiceHelper} from '../../Infrastructure/Utilities/ServiceHelper';
import {SessionLocator} from '../../Infrastructure/Utilities/SessionLocator';
import {Output, EventEmitter}  from '@angular/core';
import {PropertyChangedArgs} from '../../Infrastructure/EventEmitterArgs/PropertyChangedArgs';

export class AgentSharedLogisticsKey {
    public SharedKey: string;
    public Agent1Tenant: number;
    public Agent2Tenant: number;
    public CreateDate: Date;
    public CreatedByUserEmail: string;
    public ApprovedByUserEmail: string;
    public ApproveDate: Date;
    public InactiveDate: Date;  
    public InactiveByUserEmail: string;
    public StatusCode: string;
}

File: AmitalCloud/AmitalCloud.Infrastructure.Domain/EntityPOCOs/Manual/AgentSharedLogisticsKey.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace AmitalCloud.Infrastructure.Domain.EntityPOCOs
{
    public class AgentSharedLogisticsKey
    {
        [Key]
        public string SharedKey { get; set; }
        public int Agent1Tenant { get; set; }
        public int Agent2Tenant { get; set; }
        public DateTime CreateDate { get; set; }
        public string CreatedByUserEmail { get; set; }
        public string ApprovedByUserEmail { get; set; }
        public DateTime? ApproveDate { get; set; }

        public DateTime? InactiveDate { get; set; }
        public string InactiveByUserEmail { get; set; }

        public string StatusCode { get; set; }

    }
}
